tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -167,11 +168,13 @@ def __new__(cls,
               minimum_segment_size=3,
               maximum_cached_engines=1,
               use_calibration=True,
-              allow_build_at_runtime=True):
+              allow_build_at_runtime=True,
+              use_explicit_precision=False):
```

Comment:
We will discuss the naming of this parameter at our meeting tomorrow.
Please add document for this new parameter to the document block above.
Also, it would be nice to split this change to a separated PR, and needed update to the compatibility tests related to the API.

Reply:
ok we also recently discussed not using this specific API, I'll remove API changes

Reply:
Per offline discussion, TF-TRT will decide whether to use use_explicit_precision based on TensorRT version.